actions/publish-pages/action.yml

```diff
@@ -8,11 +8,14 @@ inputs:
   output_path:
     description: 'The path, within the github pages branch, to put the documentation.'
     required: false
-    default: './'
+    default: '.'
```

Comment:
This isn't ideal, and I think we could make the script better. The problem if the grep checking if files need to be removed. It was made with the idea of a sub-directory in mind, as we have with a monorepo.